### PR TITLE
Aditi

### PR DESCRIPTION
**Pull request by aditi2906**
Pull request title

changes in content and code

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -1231,3 +1231,11 @@
...
   "skipCi": true
 }
+"login": "aditi2906",
+      "name": "Aditi Tripathi",
+      "avatar_url": "https://avatars2.githubusercontent.com/u/61978541?v=4",
+      "profile": "https://github.com/aditi2906",
+      "contributions": [
+        "code",
+        "content"}
+
```
Comment (ohbus) [inline .all-contributorsrc:1241]: No need to add this.
Our bot will handle this once your PR gets merged into the codebase.

File: abstract-factory/src/main/java/com/iluwatar/abstractfactory/Army.java [hunk 2]
```diff
@@ -28,5 +28,5 @@
  */
 public interface Army {
 
-  String getDescription();
+   String getDescription();
```
Comment (ohbus) [inline abstract-factory/src/main/java/com/iluwatar/abstractfactory/Army.java:31]: No need for a change here
This file is formatted in two space for each tab.

File: abstract-factory/src/main/java/com/iluwatar/abstractfactory/ElfArmy.java [hunk 3]
```diff
@@ -28,7 +28,7 @@
  */
 public class ElfArmy implements Army {
 
-  static final String DESCRIPTION = "This is the Elven Army!";
+  static final String DESCRIPTION = "This is the Eleven Army!";
```
Comment (ohbus) [inline abstract-factory/src/main/java/com/iluwatar/abstractfactory/ElfArmy.java:31]: It is Elven and not Eleven.
The context here is not a number.

File: abstract-factory/src/main/java/com/iluwatar/abstractfactory/Kingdom.java [hunk 4]
```diff
@@ -42,7 +42,7 @@ public Army getArmy() {
   }
 
   public void setKing(King king) {
-    this.king = king;
+    this.king = King;
```
Comment (ohbus) [inline abstract-factory/src/main/java/com/iluwatar/abstractfactory/Kingdom.java:45]: This is violating the principles of writing variables and methods.

Find the relevant information in our [developer wiki](https://github.com/iluwatar/java-design-patterns/wiki)

File: abstract-factory/src/main/java/com/iluwatar/abstractfactory/OrcKingdom.java [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package com.iluwatar.abstractfactory;
+
+
+public interface OrcKingdom {
+
+  String getDescription();
+}
```
Comment (ohbus) [inline abstract-factory/src/main/java/com/iluwatar/abstractfactory/OrcKingdom.java:29]: Add a line after a file ends

File: abstract-factory/src/main/java/com/iluwatar/abstractfactory/OrcKingdomFactory.java [hunk 6]
```diff
@@ -42,4 +42,8 @@ public King createKing() {
   public Army createArmy() {
     return new OrcArmy();
   }
+  @Override
+  public Kingdom createKingdom(){
+    return new OrcKingdom();
+  }
```
Comment (ohbus) [inline abstract-factory/src/main/java/com/iluwatar/abstractfactory/OrcKingdomFactory.java:48]: Please explain this in the PR description.

File: event-queue/src/main/java/com/iluwatar/event/queue/Audio.java [hunk 7]
```diff
@@ -135,7 +135,7 @@ public void playSound(AudioInputStream stream, float volume) {
   private void update() {
     // If there are no pending requests, do nothing.
     if (headIndex == tailIndex) {
-      return;
+      return null;
```
Comment (ohbus) [inline event-queue/src/main/java/com/iluwatar/event/queue/Audio.java:138]: @iluwatar your comments on this please.

File: event-queue/src/main/java/com/iluwatar/event/queue/Audio.java [hunk 8]
```diff
@@ -171,7 +171,7 @@ public AudioInputStream getAudioStream(String filePath)
    * @return PlayMessage[]
    */
   public PlayMessage[] getPendingAudio() {
-    return pendingAudio;
+    return PendingAudio;
```
Comment (ohbus) [inline event-queue/src/main/java/com/iluwatar/event/queue/Audio.java:174]: No need to change this.
Please follow the design guidelines.

File: event-queue/src/main/java/com/iluwatar/event/queue/PlayMessage.java [hunk 9]
```diff
@@ -50,7 +50,7 @@ private void setStream(AudioInputStream stream) {
   }
 
   public float getVolume() {
-    return volume;
+    return Volume;
```
Comment (ohbus) [inline event-queue/src/main/java/com/iluwatar/event/queue/PlayMessage.java:53]: Follow the guidelines of writing Java code.

**Review by ohbus — CHANGES_REQUESTED** [review]

**Comment by iluwatar** [comment]
@aditi2906 These changes do not help. Please see https://github.com/iluwatar/java-design-patterns/wiki/01.-How-to-contribute

@ohbus thanks for reviewing

**Pull request closed without merging**